.goreleaser.yml

```diff
@@ -0,0 +1,93 @@
+project_name: cranlogs
+
+release:
+  # in case there is an indicator for this in the tag e.g. v1.0.0-rc1
+  # If set to true, will mark the release as not ready for production.
+  # Default is false.
+  prerelease: auto
+
+before:
+  hooks:
+    - go mod tidy
+    - ./scripts/completions.sh
+    - ./scripts/manpages.sh
+
+builds:
+  - 
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - windows
+      - darwin
+      - linux
+    goarch:
+    - amd64
+    - arm64
+    - arm
+    goarm:
+    - "7"
+
+archives:
+  - name_template: '{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    replacements:
+      darwin: Darwin
+      linux: Linux
+      windows: Windows
+      amd64: x86_64
+    format_overrides:
+    - goos: windows
+      format: zip
+    files:
+      - README.md
+      - LICENSE
+      - completions/*
+      - manpages/*   
+
+brews:
+  # Repository to push the tap to.
+  -
+    tap:
+      owner: devOpifex 
+      name: homebrew-tap 
+    folder: Formula
+    goarm: "7"
+    test: |
+      system "#{bin}/cranlogs -v"
+    install: |-
+      bin.install "cranlogs"
+      bash_completion.install "completions/cranlogs.bash" => "cranlogs"
+      zsh_completion.install "completions/cranlogs.zsh" => "_cranlogs"
+      fish_completion.install "completions/cranlogs.fish"
+      man1.install "manpages/cranlogs.1.gz"
+
+publishers:
+  - name: fury.io
+    ids:
+    - packages
+    env:
+    - 'FURY_TOKEN={{ .Env.FURY_TOKEN }}'
+    cmd: ./scripts/fury-upload.sh {{ .ArtifactName }}
+
+nfpms:
+  - file_name_template: '{{ .ConventionalFileName }}'
+    id: packages
+    homepage:  https://github.com/devOpifex/cranlogs
+    description: Access cranlogs from the command line 
+    maintainer: John Coene <john@opifex.org>
+    license: MIT
+    contents:
+      - src: ./completions/cranlogs.bash
+        dst: /etc/bash_completion.d/cranlogs
+      - src: ./completions/cranlogs.fish
+        dst: /usr/share/fish/completions/cranlogs.fish
+      - src: ./completions/cranlogs.zsh
+        dst: /usr/local/share/zsh/site-functions/_cranlogs
+      - src: ./manpages/cranlogs.1.gz
+        dst: /usr/share/man/man1/cranlogs.1.gz
+    formats:
+    - apk
+    - deb
+    - rpm
+
+snapshot:
```

Comment:
when goreleaser runs via `--snapshot` how it should name the snapshot binary - default is some incantation with the git commit. This will instead just grab the latest tag + `-next`